app/src/main/java/com/example/fittracker/mvp/view/UserSettingsView.java

```diff
@@ -1,8 +1,10 @@
 package com.example.fittracker.mvp.view;
```

Comment:
Why do you have a TextView and an EditText for each field?  Use only an EditText